generators/client/templates/angular/package.json.ejs

```diff
@@ -96,6 +96,7 @@ Remove it once the issue is closed
     "chai-as-promised": "7.1.1",
     "chai-string": "1.5.0",
     <%_ } _%>
+    "circular-dependency-plugin": "5.0.2",
```

Comment:
I believe that the circular dependency issues should be resolved after removing barrel files. Do we still require this plugin? Did you find any circular dependency issues in code after removing barrels?  I will prefer to keep build slim and not include by default unless it provides a value add.

Reply:
No, after removing barrels there is no circular dependencies any more.  
This plugin detects circular dependencies. If someone makes change which introduces circular dependency then this plugin catches this. So this plugin ensures code quality.  
But I can remove this. Should I remove this?

Reply:
yes, lets remove from default configuration. We can add such recommendations on [tips](https://www.jhipster.tech/tips/) page

Reply:
Ok, I will remove this.

Reply:
circular-dependency-plugin removed now.